pisek/config/config_description.py

```diff
@@ -39,14 +39,28 @@ def __init__(
         self.value = value
         super().__init__()
 
+    @abstractmethod
+    def _op(self, current_value) -> bool:
+        pass
+
     def check(self, section: str, config: "ConfigHierarchy") -> str:
         section = self.section.transform_name(section)
         current_value = self.key.get(config, section=section)
-        if current_value == self.value:
+        if self._op(current_value):
             return ""
         return f"[{section}] {self.key.key}={current_value}\n"
 
 
+class HasKeyValue(KeyValueCond):
```

Comment:
I'm not sure what this name is supposed to mean. Wouldn't `KeyHasValue` make more sense, if that's what you mean?